mne/tests/test_cov.py

```diff
@@ -496,6 +497,17 @@ def test_cov_scaling():
     _undo_scaling_array(data, picks_list, scalings=scalings)
     assert_allclose(data, evoked.data, atol=1e-20)
 
+    # check that scaling is undone
+    raw = read_raw_fif(raw_fname).crop(0, 5).load_data()
+    events = find_events(raw, stim_channel='STI 014')
+    raw.pick_channels(raw.ch_names[:5])
+    raw.add_proj([], remove_existing=True)
+    epochs = Epochs(raw, events, None, tmin=-0.2, tmax=0., preload=True)
+    data = np.hstack(epochs.get_data())
+    data_orig = data.copy()
+    _ = _regularized_covariance(data, info=epochs.info)
```

Comment:
you don't need to have `_ = ` here, this is why Travis complains

https://travis-ci.org/mne-tools/mne-python/jobs/451425389#L3283

Reply:
I've been looking into it further, and there's no need for `info`. So we don't need to read anything.

The following also fails in master and passes here:
```py
    # check that scaling is undone
    # raw = read_raw_fif(raw_fname).crop(0, 5).load_data()
    # events = find_events(raw, stim_channel='STI 014')
    # raw.pick_channels(raw.ch_names[:5])
    # raw.add_proj([], remove_existing=True)
    # epochs = Epochs(raw, events, None, tmin=-0.2, tmax=0., preload=True)
    # data = np.hstack(epochs.get_data())
    _regularized_covariance(data)
    assert_array_almost_equal(data, evoked.data)
```

But the timing is much better, see this:
```sh
Test session starts (platform: linux, Python 3.6.6, pytest 3.6.3, pytest-sugar 0.9.1)
rootdir: /home/sik/code/mne-python, inifile: setup.cfg
plugins: smother-0.2, sugar-0.9.1, pudb-0.6, ipdb-0.1, faulthandler-1.5.0, cov-2.5.1

 mne/tests/test_cov_copy.py ✓✓                                                            100% ██████████
======================================= slowest 20 test durations =======================================
0.23s call     mne/tests/test_cov_copy.py::test_cov_scaling_READING
0.14s call     mne/tests/test_cov_copy.py::test_cov_scaling_WITHOUT_READING
0.00s setup    mne/tests/test_cov_copy.py::test_cov_scaling_READING
0.00s teardown mne/tests/test_cov_copy.py::test_cov_scaling_READING
0.00s teardown mne/tests/test_cov_copy.py::test_cov_scaling_WITHOUT_READING
0.00s setup    mne/tests/test_cov_copy.py::test_cov_scaling_WITHOUT_READING
```

Reply:
Let's do this later and move toward merge now.